intersection_transit_maneuvering/src/itm_service.cpp

```diff
@@ -0,0 +1,27 @@
+#include "itm_service.h"
+#include <ros/ros.h>
+#include <vector>
+#include <cav_msgs/TrajectoryPlan.h>
+#include <cav_msgs/TrajectoryPlanPoint.h>
+#include <cav_msgs/Plugin.h>
+#include <boost/shared_ptr.hpp>
+#include <carma_utils/CARMAUtils.h>
+#include <boost/geometry.hpp>
+#include <carma_wm/Geometry.h>
+#include <cav_srvs/PlanTrajectory.h>
+namespace itm_servicer
+{
+
+    Servicer::Servicer(){};
+
+    bool Servicer::call(cav_srvs::PlanTrajectoryRequest& req, cav_srvs::PlanTrajectoryResponse& resp)
+    {
+        client.call(req,resp);
```

Comment:
This method returns bool but no return call is made. Presumably you want 
return client.call